features/orders.feature

```diff
@@ -8,10 +8,10 @@ Feature: List orders
     And I'm loged in as admin user "admin@example.com"
     And I go to the dashboard
     And the following orders exist:
-    | name       | description | price | order_date       | delivery_date    | email                  |
-    | Bob Schmob | Bobs order  | 100   | 2017-11-15 10:00 | 2017-11-20 12:00 | bob.schmob@example.com |
-    | John Smith | Johns order | 200   | 2017-10-05 11:00 | 2017-10-10 12:00 | john.smith@example.com |
-    | Kari Lee   | Karis order | 300   | 2017-10-03 14:00 | 2017-10-08 12:00 | kari.lee@example.com |
+    | billing_name             | delivery_date    | billing_email          |
```

Comment:
why so tabing?

Reply:
I don't understand the comment. The table was changed to the model changes.

Reply:
I looks like some extra tabing

Reply:
This is a table with test data. Don't understand the comment.

Reply:
No drama guys. Looks good. Hard to see on GitHub, that's all